src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Java/JavaMarshal.cs

```diff
@@ -3,13 +3,18 @@
 
 using System.Runtime.CompilerServices;
 using System.Runtime.Versioning;
+using System.Threading;
 
 namespace System.Runtime.InteropServices.Java
 {
     [CLSCompliant(false)]
     [SupportedOSPlatform("android")]
     public static partial class JavaMarshal
     {
+#if !NATIVEAOT
+        private static Thread? s_bridgeThread;
+#endif
+
```

Comment:
```suggestion
```
Nit: It does not need to be stored in a static.

Reply:
As in, just allocate it and don't worry about it?

Reply:
Right, allocate it, start it and don't worry about it.